index.js

```diff
@@ -137,8 +136,7 @@ module.exports = ({consumerKey, consumerSecret, accessToken, accessTokenSecret,}
         const t = new Twit({
             consumer_key: consumerKey,
             consumer_secret: consumerSecret,
-            access_token: requestToken,
```

Comment:
This one should be unchanged. After authenticating as a user, you need the request token and request token secret to retrieve their access token.